lib/rules/no-todo.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @fileoverview Forbid the use of QUnit.todo
+ * @author Baptiste Doucerain
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const utils = require("../utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
```

Comment:
There's a new requirement on master to add this comment:

```suggestion
/** @type {import('eslint').Rule.RuleModule} */
module.exports = {
```

Reply:
Ok

Reply:
You can just accept the suggestion to fix this :)

Reply:
Done